RELEASE_NOTES

```diff
@@ -1,3 +1,11 @@
+9.3.0
+=====
+
+1. Fix FQDN events.
+   Limitations:
+     - this fix uses a dirty trick which breaks backward compatibility with notifd;
```

Comment:
Actually this is not really true. Events are still working with notifd, even with this version.
The only feature which might not work is the event filtering feature from the notifd events, but it seems like this feature is no longer supported since Tango 8 (http://tango-controls.readthedocs.io/en/latest/development/client-api/cpp-client-programmers-guide.html?highlight=Event%20filtering#event-filtering-removed-in-tango-release-8-and-above). So we are good here it seems ;-).

Reply:
OK, good to know :) Then we have to _really_ remove them at some point...